docs/usage/cli.md

```diff
@@ -43,10 +43,10 @@ bin/uc catalog create --name my_catalog --comment "My First Catalog"
 
 ### Update a Catalog
 ```sh
-bin/uc catalog update --name <name> --new_name <new_name> [--comment <comment>]
+bin/uc catalog update --name <name> [--new_name <new_name>] [--comment <comment>]
 ```
 - `name` : The name of the existing catalog.
-- `new_name` : The new name of the catalog.
+- `new_name` : *[Optional]* The new name of the catalog.
 - `comment` : *[Optional]* The new description of the catalog.
```

Comment:
Could we please add a comment here mentioning that at least one optional parameter is needed?